### ISSUE TITLE
blocklist support

### ISSUE THREAD

**Issue by theoephraim** [webtorrent/webtorrent-desktop#1190]
Just a feature request for blocklist support!
Looks like the underlying cli supports it, so its just a matter of adding it to the preferences and maybe some kind of downloading/caching?

**Comment by feross** [comment]
Yep, we ought to expose this (and probably hide it under an Advanced section of the preferences)
